src/lib/components/chat/ChatIntroduction.svelte

```diff
@@ -19,6 +20,11 @@
 
 	$: currentModelMetadata = findCurrentModel(models, settings.activeModel);
 
+	let announcementBanners = JSON.parse(PUBLIC_ANNOUNCEMENT_BANNERS);
+	let title = announcementBanners[0].title;
+	let linkTitle = announcementBanners[0].linkTitle;
+	let linkHref = announcementBanners[0].linkHref;
```

Comment:
We only use the first value here, while several announcement banners can be stored in the environment variable.

We should be able to display multiple announcement banners if there are multiple in the environment variable. Or  none if the variable is an empty array (or is empty)

Reply:
I see of course, thanks for clearing that up for me.